asterius/rts/rts.unicode.mjs

```diff
@@ -29,7 +29,7 @@ export class Unicode {
     }
 
     u_towtitle(code){
-        return u_toupper(code);
+        return this.u_towupper(code); //FIXME
```

Comment:
Would be good to open a related ticket for this one, or all future FIXME or TODO comments.

Reply:
I have opened https://github.com/tweag/asterius/issues/274